test/test-apimatic-core/utilities/json_pointer_helper_test.rb

```diff
@@ -0,0 +1,53 @@
+require 'minitest/autorun'
+require 'apimatic_core'
+
+class JsonPointerHelperTest < Minitest::Test
+  include CoreLibrary
+  def setup
+    @hash = {
+      water: ['river', 'lake', 'ocean', 'pond', 'everything else'],
+      fire: {
+        water: {
+          wind: 'earth'
+        },
+        dirt: [
+          { foo: 'bar', hello: 'world' },
+          { baz: 'biz' }
+        ]
+      }
+    }
+  end
+
+
+  def test_exists
+    # Test cases for #exists? method
+    assert_pointer_exists(JsonPointerHelper.new(@hash, '/fire/dirt/0/hello', symbolize_keys: true))
+    assert_pointer_exists(JsonPointerHelper.new(@hash, '/water/~/dirt', symbolize_keys: true), false)
```

Comment:
Nits, Blocker: `assert_pointer_not_exists` method should be used here instead of passing the `exists` argument value as false.